src/utils/container.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Checks if the list includes any value from the search.
+ * @param list The list to search in.
+ * @param search The values to search.
+ * @returns {boolean} _true_ if the list includes any value from the search.
+ */
+export function includesAny(list: string | string[] | null | undefined, search: string | string[]) {
+    if (!list) {
+        return true;
+    }
+
+    let negative = false;
+
+    if (typeof list === 'string') {
+        if (list.startsWith('-')) {
+            negative = true;
+            list = list.substring(1);
+        }
+
+        list = list.split(',');
+    }
+
+    list = list.filter(i => i);
```

Comment:
I'm not sure what this actually does? :sweat_smile:

Reply:
Removes empty items